src/xx-apt

```diff
@@ -139,6 +139,15 @@ if [ "$TARGETARCH" = "riscv64" ] && [ "$(xx-info vendor)" = "debian" ]; then
   apt-get install -y debian-ports-archive-keyring
   echo "deb [ arch=riscv64 ] http://ftp.ports.debian.org/debian-ports sid main" >>/etc/apt/sources.list.d/riscv64-sid.list
 fi
+if [ "$TARGETARCH" = "loongarch64" ] && ( [ "$(xx-info vendor)" = "debian" ] || [ "$(xx-info vendor)" = "Loongnix" ] ); then
+  apt-get update
+  apt-get install -y debian-ports-archive-keyring
+  if [ -z "$XX_APT_REPO_URL_LOONGARCH64" ]; then
+  echo "Please set the environment variable XX_APT_REPO_URL_LOONGARCH64=pkg.loongnix.cn"
+  exit 0;
+  fi
+  echo "deb [ arch=loongarch64 ] http://pkg.loongnix.cn/loongnix  DaoXiangHu-stable main contrib non-free" >>/etc/apt/sources.list.d/loongarch64.list
```

Comment:
Should use the env as well. You can put the distro in another env or combine them somehow.